BringgSDK.js

```diff
@@ -673,6 +682,48 @@ var BringgSDK = (function () {
         : REAL_TIME_OPTIONS.SECURED_SOCKETS;
   }
 
+  /** get region name from region code number */
+  module.getRegionByCodeNumber = function (regionCode) {
+    if (!regionCode) {
+      log('getRegionByCodeNumber: no region code was provided');
+      return;
+    }
+
+    if (typeof regionCode !== 'number') {
+      log('getRegionByCodeNumber: region code must be a number');
+      return;
+    }
+
+    for (var regionName in REGIONS_ENUM) {
+      if (REGIONS_ENUM[regionName] === regionCode) {
+        return regionName;
+      }
+    }
+  };
+
+  /** get region code from shared url */
+  module.getRegionCodeFromUrl = function (url) {
+    var regionCode = url.split('&e=')[1];
+    return parseInt(regionCode);
```

Comment:
parseInt can return a NaN upon failure.
sadly typeof NaN === number apparently so the check you did in the method above won't hold.
so please add an explicit check to either NaN or wrap the parseInt somehow

Reply:
I have the check in `module.getRegionByCodeNumber` for parseInt and you can see in the unit test also

```
if (typeof regionCode !== 'number') {
      log('getRegionByCodeNumber: region code must be a number');
      return;
}

```